commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java

```diff
@@ -453,13 +457,20 @@ private InfoResponseDto mergeProcessWithMultipleIteration(InfoResponseDto infoRe
     private ContainerInfoDto setContainerInfo(List<ContainerInfoDto> finalInfos, ContainerInfoDto info, String process) {
         ContainerInfoDto finalInfo = finalInfos.stream().filter(
                 i -> i.getSource().equals(info.getSource()) && i.getProcess().equals(process)).findAny().get();
```

Comment:
findAny() returns an optional object here. Instead of directly calling get(), extract it to Optional<ContainerInfoDto>, check using isPresent() method and then do a get()